string crazyflieName
---
CrazyflieContext crazyflieContext
uint32 studentID